prow/config.yaml

```diff
@@ -91,7 +91,7 @@ presubmits:
     - master
     spec:
       containers:
-      - image: gcr.io/istio-testing/prowbazel:0.3.3
+      - image: gcr.io/istio-testing/prowbazel:0.3.4
```

Comment:
gitub doesn't seem to let me comment on an unchanged line.  There's two instance of 0.3.2 in this file that I'm guessing aren't at that version for a particular reason so they could also be updated to 0.3.4 (to keep things consistent).

Reply:
fixed